src/Wellcome.Dds/Wellcome.Dds.Server/Startup.cs

```diff
@@ -131,6 +140,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.SetupSwagger();
             app.UseStaticFiles();
             app.UseRouting();
+            app.UseAuthorization();
```

Comment:
Does this add any other overhead?

Reply:
Unscientific testing locally there seems to be a slight hit initially but once everything is bootstrapped it's not. Made 4 requests with + without:
```bash
# with
1649ms -> 63ms -> 13ms -> 15ms

# without
1357ms -> 43ms -> 15ms -> 14ms
```